README.md

```diff
@@ -323,7 +328,7 @@ queue = require 'queue'
 The request "require 'queue'" causes automatic creation of
 the `_queue` space, unless it already exists. The same request
 also sets the necessary space triggers and other objects
-associated with queues.  
```

Comment:
Why did you decide to remove the double space ("\n" in md) here and below?
These are unnecessary changes. If you want to update something that not related with patchset, create a separate PR.
I suggest you read https://github.com/tarantool/tarantool/wiki/Code-review-procedure#general-coding-points-to-check and https://github.com/tarantool/tarantool/wiki/Code-review-procedure#commit-message .

Reply:
I got your point on including only relevant updates into PR.

If these double spaces serve some purpose and are not just typos, I will correct this.

Reply:
Two whitespaces at the end of line [issues](https://daringfireball.net/projects/markdown/syntax#p) a `<br />` tag into HTML. Personally I consider this feature as broken in two ways: the syntax is weird, the feature is not needed (it always better to just split text with paragraphs or use three-backticks code blocks). So I prefer to don't use it. And GFM (GitHub's Markdown dialect) anyway preserves line breaks, so it is useless on GitHub.

Anyway, Leonid is right, it is not relevant to the problem that you are solving.